alpha/apps/kaltura/lib/myPlaylistUtils.class.php

```diff
@@ -206,7 +206,8 @@ public static function executePlaylist ($partner_id , $playlist , $filter = null
 				break;
 			case PlaylistType::STATIC_LIST:
 			case PlaylistType::PATH:
-				$entryObjectsArray = self::executeStaticPlaylist ( $playlist , $filter , $detailed, $pager );
+                		$entry_id_list_str = $playlist->getDataContent();
+				$entryObjectsArray = self::executeStaticPlaylistFromEntryIdsString($entry_id_list_str, $filter, $detailed, $pager, $playlist->getEntryId());
```

Comment:
I would rename $entry_id_list_str to $entryIdList to keep the naming convention.